__tests__/app.test.js

```diff
@@ -76,4 +75,49 @@ describe('endpoint GET /api/articles/:article_id',()=>{
             expect(body.msg).toBe('404: Article not found')
         })
     })
-})
\ No newline at end of file
+})
+
+describe('endpoint GET /api/articles/:article_id/comments',()=>{
+    test('receives 200 response and commments for given article_id',()=>{
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then(({body})=>{
+            const date = new Date(1600560600000)
+            expect(body.comments).toEqual(  [{
+                comment_id: 10,
+                body: "git push origin master",
+                votes: 0,
+                author: "icellusedkars",
+                article_id: 3,
+                created_at: "2020-06-20T07:24:00.000Z",
+              },
+              {
+                comment_id: 11,
+                body: "Ambidextrous marsupial",
+                votes: 0,
+                author: "icellusedkars",
+                article_id: 3,
+                created_at: "2020-09-19T23:10:00.000Z",
+              }])
+        })
```

Comment:
It is worth making an assertion about the order of the data - it will be useful in front end if these come back in descending date order.